### ISSUE TITLE
How to contribute

### ISSUE THREAD

**Issue by frob** [guumaster/homebrew-tap#4]
The file says that it comes from GoReleaser and not to edit it but then how do we go about fixing issues in the file. Is there another template that we should contribute to instead?

**Comment by guumaster** [comment]
The fix should come from `goreleaser` probably

**Comment by frob** [comment]
Yeah, looks like something to do with https://goreleaser.com/customization/universalbinaries/ I am not a go developer so I don't really know what additions would need to be made but it looks like this feature only came out a couple of months ago.

After looking through all this I might pick go up. The toolchain looks really appealing.
